src/utils/identity/secpk256k1/identity.ts

```diff
@@ -111,8 +111,8 @@ class Secp256k1KeyIdentity extends GenericSignIdentity {
   /**
    * Serialize this key to JSON.
    */
-  public toJSON(): string {
-    return JSON.stringify([blobToHex(this._publicKey.toRaw()), blobToHex(this._privateKey)]);
+  public toJSON(): [string, string] {
```

Comment:
What u think if we create a type JSONABLEKeyPair = [string, string] or something like that?